ew/utils/loop.py

```diff
@@ -858,6 +859,87 @@ async def enemy_action_tick_loop(id_server):
             ewutils.logMsg('Enemy tick loop failed. Reason:{}'.format(e))
 
+severityLimit = {
+    "low": 5,
+    "medium" : 15,
+    "high" : 25,
+    "wasted" : 50,
+    "wrecked" : 90,
+    "totalled" : 100
+}
+async def drunk_effects(id_server):
+    while not ewutils.TERMINATE:
+        timer = 30
+        await asyncio.sleep(timer)
+        users = bknd_core.execute_sql_query("SELECT inebriation, id_user FROM users WHERE inebriation >0")
+        server = ewcfg.server_list[id_server]
+        if users:
+            for inebriation, user in users:
+                user_data = EwUser(id_server=id_server, id_user=user)
+                player = EwPlayer(id_user=user)
+                inebriation=user_data.inebriation
+                mutations = user_data.get_mutations()
+                channel = fe_utils.get_channel(server,user_data.poi)
+                
+                chance = max(1, 50-inebriation)
+                #More frequently, the more drunk
+                if random.randint(1,chance)==1:
+                    user_data = EwUser(id_server=id_server, id_user=user)
+                    injury_text = ""
+                    slimeloss=0
+                    slime = user_data.slimes
+                    
+                    savingthrow = random.randint(1,20)
+                    # Lucky rolls with advantage
+                    if ewcfg.mutation_id_lucky in mutations:
+                        savingthrow = max(random.randint(1,20),random.randint(1,20)) 
+
+                    if inebriation<=severityLimit.get("low"):
+                        injury_text=["{name} stumbles around.".format(name=player.display_name),"{name} slurs their words as they attempt to speak.".format(name=player.display_name),"{name} burps.".format(name=player.display_name)]
+                        
+                    elif inebriation<=severityLimit.get("medium"):
+                        if savingthrow > 10:
+                            injury_text = ["{name} tries to stay balanced... they succeed!".format(name=player.display_name),"{name} belches.".format(name=player.display_name)]       
+                        else:
+                            slimeloss = min(50,slime)
+                            injury_text = ["{name} tries to stay balanced but fails, falling to the ground, catching a few grazes on the arm they used to brace. **They lose {slime} slime!**".format(name=player.display_name, slime=slimeloss),"{name} Walks into a lamppost. **They lose {slime} slime!**".format(name=player.display_name, slime=slimeloss)]
+                                
+                        
+                    elif inebriation<=severityLimit.get("high"):
+                        if savingthrow > 12:
+                            injury_text = ["{name} begins to tilt over, before they can fall and faceplant the pavement they catch themselves on a railing. Nice save!".format(name=player.display_name),"{name} gurks.".format(name=player.display_name)]
```

Comment:
You need another word to join two independent clauses, the comma ain't enough. Probably "though"?
"can fall and" is kind of not needed, "faceplant" implies that the user is falling.